Orange/widgets/data/owsave.py

```diff
@@ -69,7 +76,7 @@ def save_file_as(self):
             filename += format_extensions[0]
         self.filename = filename
         self.last_dir, file_name = os.path.split(self.filename)
-        self.save.setText("Save as '%s'" % file_name)
+        self.info.setText("Saving as '%s'" %  file_name)
         self.save.setDisabled(False)
         self.save_file()
```

Comment:
`gui.auto_commit` assigns a wrapper to `self.save_file`. In this line, you must call `self.unconditional_save_file`, otherwise "Save as..." will only save if auto save is on. The other call (in function `self.dataset`) is OK, because there you want conditional saving.